src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -240,10 +236,9 @@ protected void validateAdmission(Admission admission, boolean insert) throws OHS
 		List<Admission> admList = getAdmissions(patient);
 
 		/*
-		 * Today Gregorian Calendar
+		 * Today LocalDateTime
 		 */
-		GregorianCalendar today = TimeTools.getDateToday24();
-		DateFormat currentDateFormat = DateFormat.getDateInstance(DateFormat.SHORT, new Locale(GeneralData.LANGUAGE));
+		LocalDateTime today = LocalDateTime.now();
```

Comment:
The hotfix here was:
```
-		GregorianCalendar today = new GregorianCalendar();
+		GregorianCalendar today = TimeTools.getDateToday24();
```

So it seems the new `LocalDateTime today = LocalDateTime.now();` is rolling back the hotfix, isn't it?

Reply:
yep; fixed it; thanks